Automattic/jetpack
# E2E tests: run Atomic tests on weekly releases

#### Changes proposed in this Pull Request:

* Add a functionality plugin that adds API endpoint to manually trigger Jetpack beta autoupdates
* Add a .cjs script to trigger and wait for the site to be updated to the latest version
* Add `atomicRc` site to secrets that is used for WoA RC tests

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






The script could be tested manually against a test site.
* For this, its configuration with the app password should be added to the config file. 
* Add `e2e-beta-autoupdate-api.php` plugin to the site
* Make sure Jetpack Beta plugin is active and Jetpack is running RC version from the Beta plugin
* Downgrade plugin version manually in `.jpbeta.json` and `.jetpack.php` in `jetpack-dev` plugin
* run `TEST_SITE="SITE_NAME" node ./bin/wait-for-atomic-site.cjs` and check the output. 
* Check the Jetpack's version on your site. 

The workflow bits could be tested only after merge.
